app/src/main/java/hr/dice/filipbionda/tmdbpractice/ui/homescreen/HomeScreen.kt

```diff
@@ -0,0 +1,558 @@
+                            fontSize = 16.sp,
+                            color = MaterialTheme.colorScheme.surfaceBright,
+                        ),
+                    )
+                },
+                trailingIcon = {
+                    if (expanded) {
+                        IconButton(
+                            onClick = {
+                                if (query.isEmpty()) {
+                                    onExpandedChange(false)
+                                } else {
+                                    query = ""
+                                }
+                            },
+                        ) {
+                            Icon(
+                                imageVector = Icons.Default.Clear,
+                                contentDescription = null,
+                            )
+                        }
+                    } else {
+                        Icon(
+                            imageVector = Icons.Default.Search,
+                            contentDescription = null,
+                            tint = MaterialTheme.colorScheme.surfaceBright,
+                        )
+                    }
+                },
+            )
+        },
+        expanded = expanded,
+        onExpandedChange = {
+            onExpandedChange(it)
+        },
+    ) {
+        LazyColumn(
+            modifier = Modifier.padding(top = 10.dp),
+        ) {
+            items(suggestions) { suggestion ->
+                Row(
+                    modifier = Modifier.padding(horizontal = dimensionResource(R.dimen.search_bar_suggestions_top_padding)),
+                ) {
+                    Icon(
+                        imageVector = Icons.Default.History,
+                        tint = MaterialTheme.colorScheme.onSurface,
+                        contentDescription = null,
+                    )
+                    Spacer(
+                        modifier = Modifier.width(dimensionResource(R.dimen.search_bar_suggestion_content_padding)),
+                    )
+                    Text(
+                        text = suggestion,
+                        style = MaterialTheme.typography.bodySmall,
+                    )
+                }
+            }
+        }
+    }
+}
+
+@Composable
+fun HomeScreenChipGroup(
+    categories: List<ContentType>,
+    onChipClick: (ContentType) -> Unit,
+    modifier: Modifier = Modifier,
+    initialCategory: ContentType = ContentType.MOVIE,
+) {
+    var selectedChip by rememberSaveable {
+        mutableStateOf(initialCategory)
+    }
+    LazyRow(
+        verticalAlignment = Alignment.CenterVertically,
+        modifier = modifier,
+    ) {
+        items(categories) { category ->
+            FilterChip(
+                modifier = Modifier
+                    .padding(
+                        start =
+                        if (category == initialCategory || category == categories.last()) {
```

Comment:
Instead of that you can set the initial padding with a spacer and spacing between the chips in the Lazy Row with  `horizontalArrangement = Arrangement.spacedBy()`

Reply:
Thanks! I had to use horizontal padding + spacing between items to get desired space between edge of the screen and Composable, but it works the same now.